tests/cc/test_brb.py

```diff
@@ -109,7 +109,7 @@ def connect_ports(self, prog_id_pem, prog_id_br, curr_pem_pid, curr_br_pid,
         br_dest_map[c_uint(curr_br_pid)] = br_dest_map.Leaf(prog_id_pem, curr_pem_pid)
         self.pem_port[c_uint(curr_pem_pid)] = c_uint(ifindex)
         self.pem_ifindex[c_uint(ifindex)] = c_uint(curr_pem_pid)
-        mac_addr = br_mac_map.Key(int(EUI(vm_mac.decode())))
+        mac_addr = br_mac_map.Key(int(EUI(vm_mac)))
```

Comment:
int(EUI(vm_mac)) won't work on python 2.7.3.
8:   File "/home/plumgrid/iovisor/bcc/tests/cc/test_brb.py", line 181, in test_brb
8:     self.config_maps()
8:   File "/home/plumgrid/iovisor/bcc/tests/cc/test_brb.py", line 136, in config_maps
8:     self.ns1_eth_out.index, self.vm1_mac, self.vm1_ip)
8:   File "/home/plumgrid/iovisor/bcc/tests/cc/test_brb.py", line 112, in connect_ports
8:     mac_addr = br_mac_map.Key(int(EUI(vm_mac())))
8: TypeError: 'str' object is not callable

This is yet another python 2/3 compatibility issue.